GoRest.Api.Client/Client/Extentions/ResponseHttpStatusCodeValidationExtention.cs

```diff
@@ -0,0 +1,57 @@
+﻿using FluentAssertions;
+using GoRest.Api.Client.Client.Models;
+using System.Net;
+
+namespace GoRest.Api.Client.Client.Extentions
+{
+    public static class ResponseHttpStatusCodeValidationExtention
```

Comment:
use Generics here and you will have 6 methods instead of 10

Reply:
Done